litmus-portal/backend/auth/pkg/server/server.go

```diff
@@ -38,8 +45,79 @@ func NewServer(cfg *Config) *Server {
 
 // Server Provide authorization server
 type Server struct {
-	Config  *Config
-	Manager *manage.Manager
+	Config      *Config
+	Manager     *manage.Manager
+	globalToken *oauth2.Token
```

Comment:
We can delete this variable here now. As we are already saving it in `User` model